kukur/logging.py

```diff
@@ -0,0 +1,24 @@
+"""Configure logging for Kukur."""
+# SPDX-FileCopyrightText: 2021 Timeseer.AI
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import logging.handlers
+
+from typing import Any, Dict
+
+
+def configure(config: Dict[str, Any], threaded=False):
+    """Configure the Python logger."""
+    log_format = "%(asctime)s %(levelname)s %(name)s %(processName)s : %(message)s"
+    if threaded:
+        log_format = "%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s"
+    if "logging" in config and "path" in config["logging"]:
+        path: str = config["logging"]["path"]
+        handler = logging.handlers.TimedRotatingFileHandler(
```

Comment:
To make this work with user intervention in case of a Windows service, the example config on Windows should contain:

```
[logging]
path = 'kukur.log'
```